pkg/dependency/icu.go

```diff
@@ -143,6 +143,7 @@ func (i ICU) createDependencyVersion(version string, release internal.GithubRele
 		SHA:             dependencySHA,
 		ReleaseDate:     release.CreatedDate,
 		DeprecationDate: "",
+		CPE:             fmt.Sprintf(`cpe:2.3:a:icu-project:international_components_for_unicode:%s:*:*:*:*:c\/c\+\+:*:*`, version),
```

Comment:
Why are we limiting this to just C and C++? Shouldn't we have a `*` there?

Reply:
ICU has separate libraries for c/c++ and java (icu4c and icu4j), we only build the c/c++ version.

Reply:
Ah I see. So then this is correct.